isaric/parsers/western-australia.toml

```diff
@@ -480,6 +483,15 @@
     { field = "antimal_cmyn", ref = "Y/N/NK", description = "Antimalarial agent on the first day of admission?" },
   ]
 
+[visit.treatment_oxygen_mask_prongs]
+  combinedType = "any"
+  fields = [
+    { fieldpattern = ".*oxygen_therapy", values = { 1 = true, 2 = false, 3 = true, 4 = true, 5 = false } },
+    { field = "overall_oxygen_cmtype___1", values = { 0 = false, 1 = true } },
+    { field = "overall_oxygen_cmtype___3", values = { 0 = false, 1 = true } },
+    { field = "overall_oxygen_cmtype___4", values = { 0 = false, 1 = true } },
+  ]
```

Comment:
@pipliggins should ".*oxygen_therapy" on row 489 be matching field pattern for *_oxygen_cmtype?

for the cmtype___ checkbox variables I think just map the positive responses